packages/coding-agent/src/core/system-prompt.ts

```diff
@@ -101,12 +102,37 @@ export function loadProjectContextFiles(): Array<{ path: string; content: string
 	return contextFiles;
 }
 
+function buildSkillsSection(skills: Skill[]): string {
+	if (skills.length === 0) {
+		return "";
+	}
+
+	const lines = [
```

Comment:
We should probably follow the "syntax" Anthropic are using for their skills tool in CC. Specifically, wrap the skills in these tags.

```
<available_skills>

</available_skills>
```